### DISCUSSION TITLE
Generate dummy inputs for session.run

### DISCUSSION THREAD

**Discussion by ramkrishna1121** [microsoft/onnxruntime#10329] [Other Q&A]
The C++ interface on Onnxruntime perf test has a way to create dummy inputs for the model through `PopulateGeneratedInputTestData`.
Is there a similar option for the python interface? Something like `session_options.generate_dummy_inputs = True`?

**Comment by snnn** [comment]
No.

I would suggest using https://github.com/mlcommons/inference for doing perf tests.
